tyk/api/response.go

```diff
@@ -0,0 +1,52 @@
+package api
+
+import "time"
+
+type DashboardAPIResp struct {
+	Status  string `json:"Status"`
+	Message string `json:"Message"`
+	Meta    string `json:"Meta"`
+}
+
+type OrgAPIResp struct {
+	Organisations []map[string]interface{} `json:"organisations"`
+	Pages         int                      `json:"pages"`
+}
+
+type GetUserResp struct {
+	APIModel        struct{}          `json:"api_model"`
+	FirstName       string            `json:"first_name"`
+	LastName        string            `json:"last_name"`
+	EmailAddress    string            `json:"email_address"`
+	Password        string            `json:"password"`
+	OrgID           string            `json:"org_id"`
+	Active          bool              `json:"active"`
+	ID              string            `json:"id"`
+	AccessKey       string            `json:"access_key"`
+	UserPermissions map[string]string `json:"user_permissions"`
+}
+
+type ListUsersResp []GetUserResp
+
+type CreateUserResp struct {
```

Comment:
Why don't we just access data we need here?

Reply:
what data can we infer from this struct?

Reply:
We are just using `Meta.ID` and `Message`. We can keep the `Status`.
But we can remove all other info from Meta. Especially `PasswordMaxDays`,`PasswordUpdated`, `PWHistory`, `CreateAt`. I don't think we will need this data.

Reply:
oh, i see now. will remove them 👍

Reply:
removed in 68de1e3d2ea0c759b76da7da935f5c61be1ec096